docs/products/metrics/howto/migrate-influxdb-thanos.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Migrate Aiven for InfluxDB to Aiven for Metrics
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Migrate your Aiven for InfluxDB databases to Aiven for Thanos Metrics using the Aiven Console migration tool.
+
+## Compatibility considerations
+
+Before migrating an InfluxDB database to Aiven for Metrics, carefully
+review your current InfluxDB setup.
+
+- **Review database setup:** Examine your InfluxDB database's data structures, storage
+  patterns, and configurations. Identify any unique features, custom settings, and
+  specific configurations.
+- **API and query language compatibility:** Thanos Metrics uses PromQL, which is
+  different from InfluxDB's InfluxQL. Update your queries to be
+  compatible with PromQL. For assistance, you can use tools such as the
+  [InfluxQL to PromQL converter](https://github.com/logzio/influxql-to-promql-converter)
```

Comment:
Probably should clarify that built-in dashboards for Aiven services ("dashboard" integration) will not require any manual changes.

Reply:
Done